File: src/ServerPulse.Frontend/src/app/modules/shared/domain/dtos/statistics/response/loadMethodStatisticsResponse.ts
export interface LoadMethodStatisticsResponse {
    getAmount: number;
    postAmount: number;
    putAmount: number;
    patchAmount: number;
    deleteAmount: number;
}
export function convertToSLoadMethodStatisticsResponse(data: any): LoadMethodStatisticsResponse {
    return {
        getAmount: data?.GetAmount,
        postAmount: data?.PostAmount,
        putAmount: data?.PutAmount,
        patchAmount: data?.PatchAmount,
        deleteAmount: data?.DeleteAmount,
    };
}

File: src/ServerPulse.Backend/AnalyzerApi.Infrastructure/Dtos/Responses/Statistics/LoadMethodStatisticsResponse.cs
namespace AnalyzerApi.Infrastructure.Dtos.Responses.Statistics
{
    public class LoadMethodStatisticsResponse : BaseStatisticsResponse
    {
        public int GetAmount { get; set; }
        public int PostAmount { get; set; }
        public int PutAmount { get; set; }
        public int PatchAmount { get; set; }
        public int DeleteAmount { get; set; }
    }
}